lib/error_tracker/integrations/oban.ex

```diff
@@ -32,35 +32,31 @@ defmodule ErrorTracker.Integrations.Oban do
 
   # https://hexdocs.pm/oban/Oban.Telemetry.html
   @events [
-    [:oban, :job, :start],
     [:oban, :job, :exception]
   ]
 
   @doc false
   def attach do
-    if Application.spec(:oban) do
+    if ErrorTracker.Integrations.Utils.application_spec(:oban) do
```

Comment:
I think this can be solved by adding an option parameter to force the attachment. It is not the perfect solution but it will avoid creating a module to encapsulate core functionality.

What do you think?

```elixir
  def attach(force_attachament \\ false) do
    if Application.spec(:oban) || force_attachament do
```

Reply:
I agree it's a lighter approach and it can be called in the setup.